extensions/amp-accordion/1.0/accordion.jss.js

```diff
@@ -39,10 +39,21 @@ const header = {
 // or delete if not used
 const content = {};
 
+const contentHidden = {
+  display: 'none !important',
+};
+
+const contentHiddenMatchable = {
+  contentVisibility: 'hidden-matchable !important',
+  display: 'block !important',
```

Comment:
How do we know that `contentHiddenMatchable {display: block !important}` will override the one above `contentHidden {display: none !important}`? I think this is fragile. I think you should either use only one of these classes when you compute the `className`, or you can do a compound selector here, which would guarantee the correct CSS specificity.

Reply:
Updated logic to use only one of these classes.